products/redhat-jboss-eap.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Red Hat JBoss Enterprise Application Platform
+category: server-app
+iconSlug: redhat
+permalink: /redhat-jboss-eap
+alternate_urls:
+-   /redhat-jboss-eap
+-   /redhatjboss
+-   /redhatjbosseap
+
+versionCommand: $JBOSS_HOME/bin/version.sh
+releasePolicyLink: https://access.redhat.com/support/policy/updates/jboss_notes#phases
+activeSupportColumn: Full Support
+eolColumn: Maintenance Support
+releaseDateColumn: true
+extendedSupportColumn: Extended Life Cycle Support
+
+releases:
+-   releaseCycle: "7"
+    support: 2023-06-30
+    eol: 2025-06-30
+    extendedSupport: 2029-11-30
+    latest: "7.4.6"
+    releaseDate: 2016-05-01
+    lts: 2026-11-30
+    latestReleaseDate: 2023-03-01
+    link: https://access.redhat.com/articles/6716841
+
+-   releaseCycle: "6"
+    support: 2016-06-30
+    eol: 2019-06-30
+    extendedSupport: 2025-06-30
+    latest: "6.4"
+    releaseDate: 2012-06-01
+    lts: 2022-06-30
+    latestReleaseDate: 2022-06-30
+    link: https://access.redhat.com/articles/3099701
+
+-   releaseCycle: "5"
+    support: 2013-11-30
+    eol: 2013-11-30
+    extendedSupport: 2019-11-30
+    latest: "5.2"
+    releaseDate: 2009-11-01
+    lts: 2019-11-30
+    latestReleaseDate: 2013-12-23
+    link: https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html/release_notes_5.2.0/index
+
+-   releaseCycle: "4"
+    support: 2011-01-31
+    eol: 2010-10-31
+    extendedSupport: 2019-11-30
+    releaseDate: 2007-06-01
+    lts: 2016-11-30
+    latestReleaseDate: 2009-12-15
+    latest: '4.3'
+    link: https://www.redhat.com/en/blog/jboss-enterprise-application-platform-43-is-here
+    
+---
+
+> Red Hat JBoss Enterprise Application Platform (JBoss EAP) is an open-source Java EE (Enterprise Edition) application server.
+> It is a commercially supported and certified version of the open-source WildFly application server.
+> JBoss EAP provides a platform for developing and deploying enterprise-level Java applications.
+
+> JBoss EAP includes various features such as support for multiple programming models, such as Servlets, JSP, EJB, JMS,
+> and others, and is compatible with multiple databases, messaging systems, and web frameworks.
+
+
+## Full Support Phase
+
+Start Date: General Availability
```

Comment:
The description is a copy / paste of https://access.redhat.com/support/policy/updates/jboss_notes#phases. This description should summarize the EOL / support policy, users can always look at the `releasePolicyLink` if they need details. Moreover some parts here cannot be understood by the users if they are not explained (or linked to other document like it is done on https://access.redhat.com/support/policy/updates/jboss_notes#phases).

Some example of things I found cryptic or unnecessary :

- _...to the published Scope of Coverage and Service Level Agreement_ : What are those ? Where can I find those documents ?
- _Container distributions of Application Services products in the ELS Life Cycle Phase will be marked "Deprecated" in the Red Hat Container Catalog._ : Is it really useful on endoflife.date ?
- _The Limited Product Life Cycle of three years is generally applied to newer product areas and rapidly changing
technologies such as development frameworks._ : What does it mean for JBoss EAP users ?

Reply:
Working on it :+1: